flask_classful.py

```diff
@@ -228,6 +236,12 @@ def proxy(**forgettable_view_args):
                         response = cls.representations[list(cls.representations.keys())[0]](
                             response, code, headers)
 
+            # If the header or code is set, regenerate the response
+            elif any(x is not sentinel for x in (code, headers)):
+                # A response can be passed into `make_response` and it will set
+                # the key appropriately
+                response = make_response(response, code, headers)
```

Comment:
@eligundry - let's just make those `None`'s. 

Because of this:

```py
In [14]: with app.app_context():
    make_response("asd", object(), {})
   ....:     
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
AttributeError: 'object' object has no attribute 'encode'
```

Looked into how `make_response` works and `None` is not like a special value that clears out status codes or headers, so we can safely use that. That was my original fear and why I recommended a sentinel. My bad, broski!